From 5fa340a54a2da4a59aad64f2660d497273122cbd Mon Sep 17 00:00:00 2001
From: graysky <therealgraysky@proton.me>
Date: Sat, 20 Aug 2022 16:39:17 -0400
Subject: [PATCH 10/13] Revert
 "makechrootpkg-don-t-delete-MAKEFLAGS-and-PACKAGER"

This reverts commit df0a85fd4cb0e116d4bf5e133fc2ffde478d6ad5.
---
 src/makechrootpkg.in | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/makechrootpkg.in b/src/makechrootpkg.in
index eea7ec8..c52ef5f 100644
--- a/src/makechrootpkg.in
+++ b/src/makechrootpkg.in
@@ -176,8 +176,9 @@ prepare_chroot() {
 
 	$install -d "$copydir"/{build,startdir,{pkg,srcpkg,src,log}dest}
 
-  for x in BUILDDIR=/build PKGDEST=/pkgdest SRCPKGDEST=/srcpkgdest SRCDEST=/srcdest LOGDEST=/logdest
-		"MAKEFLAGS='${MAKEFLAGS:-}'" "PACKAGER='${PACKAGER:-}'"
+  sed -e '/^MAKEFLAGS=/d' -e '/^PACKAGER=/d' -i "$copydir/etc/makepkg.conf"
+  for x in BUILDDIR=/build PKGDEST=/pkgdest SRCPKGDEST=/srcpkgdest SRCDEST=/srcdest LOGDEST=/logdest \
+    "MAKEFLAGS='${MAKEFLAGS:-}'" "PACKAGER='${PACKAGER:-}'"
 	do
 		grep -q "^$x" "$copydir/etc/makepkg.conf" && continue
 		echo "$x" >>"$copydir/etc/makepkg.conf"
-- 
2.44.0

